sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/ExpiredTimerInfoImpl.scala

```diff
@@ -16,21 +16,15 @@
  */
 package org.apache.spark.sql.execution.streaming
 
-import org.apache.spark.sql.streaming.{ExpiredTimerInfo, TimeMode}
+import org.apache.spark.sql.streaming.ExpiredTimerInfo
 
 /**
  * Class that provides a concrete implementation that can be used to provide access to expired
  * timer's expiry time. These values are only relevant if the ExpiredTimerInfo
  * is valid.
- * @param isValid - boolean to check if the provided ExpiredTimerInfo is valid
  * @param expiryTimeInMsOpt - option to expired timer's expiry time as milliseconds in epoch time
  */
-class ExpiredTimerInfoImpl(
-    isValid: Boolean,
-    expiryTimeInMsOpt: Option[Long] = None,
-    timeMode: TimeMode = TimeMode.None()) extends ExpiredTimerInfo {
```

Comment:
Is it a side effect? It's obvious to remove isValid, but timeMode does not seem to be obvious.

Reply:
Yea - afaict - i don't think its used anywhere. Not sure why we had it before. So just removed it as part of cleanup